aar_doc/cli.py

```diff
@@ -158,38 +158,40 @@ def parse_meta(ctx: typer.Context) -> tuple[dict, dict]:
     """
 
     meta = ctx.obj["config"]["role_path"] / "meta"
+    argument_specs = ""
 
     try:
         argument_specs_yml = list(meta.glob("argument_specs.y*ml"))[0]
-    except IndexError:
-        typer.echo("Could not find meta/argument_specs.yml")
-        raise typer.Exit(code=1)
-
-    with open(argument_specs_yml, "r") as f:
-        try:
-            argument_specs = yaml.safe_load(f)
-        except yaml.YAMLError:
-            typer.echo("Not a valid YAML file: meta/argument_specs.yml")
-            raise typer.Exit(1)
-        try:
-            argument_specs = argument_specs["argument_specs"]
-        except TypeError:
-            typer.echo("Could not read meta/argument_specs.yml")
-            raise typer.Exit(1)
+        with open(argument_specs_yml, "r") as f:
+            try:
+                argument_specs = yaml.safe_load(f)
+            except yaml.YAMLError:
+                typer.echo("Not a valid YAML file: meta/argument_specs.y[a]ml")
+            try:
+                argument_specs = argument_specs["argument_specs"]
+            except TypeError:
+                typer.echo("Could not read meta/argument_specs.y[a]ml")
+    except (UnboundLocalError, IndexError):
+        pass
 
     try:
         main_yml = list(meta.glob("main.y*ml"))[0]
     except:
-        typer.echo("Could not find meta/main.yml")
+        typer.echo("Could not find meta/main.y[a]ml")
         raise typer.Exit(code=1)
 
     with open(main_yml, "r") as f:
         try:
             main = yaml.safe_load(f)
         except yaml.YAMLError:
-            typer.echo("Not a valid YAML file: meta/main.yml")
+            typer.echo("Not a valid YAML file: meta/main.y[a]ml")
             raise typer.Exit(1)
-
+        if not argument_specs:
+            try:
+                argument_specs = main["argument_specs"]
+            except TypeError:
+                typer.echo("Could not read meta/main.y[a]ml or meta/argument_specs.y[a]ml")
```

Comment:
This result is missing a test condition. And I suspect that the containing `if` condition might have a problem, since we are declaring `argument_specs = ""`. Can you please prove my concerns wrong? :D

Reply:
> This result is missing a test condition.

The test `test_missing_arg_spec` covers this.

> And I suspect that the containing if condition might have a problem, since we are declaring argument_specs = "". Can you please prove my concerns wrong? :D

Your concerns are wrong. :) If a variable is empty in Python, `if not` is true:

```
Python 3.12.6 (main, Sep  8 2024, 13:18:56) [GCC 14.2.1 20240805] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> foo = ""
>>> if not foo:
...     print("bar")
...
bar
>>> if foo:
...     print("baz")
...
>>>
```